### ISSUE TITLE
add apeglm log fold change shrinkage

### ISSUE THREAD

**Issue by bumproo** [nf-core/differentialabundance#152]
### Description of feature

this is an alternative to the ashr method already included in the pipeline

**Comment by pinin4fjords** [comment]
IIRC I had trouble with dependency management related to this, which I didn't have time to resolve at the time. 

But this will need to be a PR on the DESeq2 module, with a new test for the new type of shrinkage.

**Comment by pinin4fjords** [comment]
Need module PR first https://github.com/nf-core/modules/issues/3913

**Comment by bumproo** [comment]
this apeglm vs ashr issue has come up again for me and I was wondering if this got worked on at the Nov 23 hackathon?

**Comment by pinin4fjords** [comment]
No, don't think so. But actually, solving this has got a lot easier, since we can now use Seqera Containers to make containers with the additional dependency. I'll try to find some time for this soon
